test/utils.test.ts

```diff
@@ -24,7 +24,12 @@ suite('Utils module', () => {
   });
 
   test('getNodeExecutable should return npm', () => {
-    let mavenPath = Utils.getNodeExecutable();
-    expect(mavenPath).equals('npm');
+    let npmPath = Utils.getNodeExecutable();
+    expect(npmPath).equals('npm');
+  });
+
+  test('getPypiExecutable should return python', () => {
+    let pypiPath = Utils.getPypiExecutable();
+    expect(pypiPath).equals('"python"');
```

Comment:
python is enclosed in double brackets and npm is not.  Is this deliberate ?

Reply:
for now yes, as test are failing without that. See if you can get it work

Reply:
I have fixed it